src/main/kotlin/org/rust/cargo/service/CargoInstallationManager.kt

```diff
@@ -1,29 +0,0 @@
-package org.rust.cargo.service
-
-import com.intellij.openapi.projectRoots.SdkType
-import org.rust.cargo.util.PlatformUtil
-import org.rust.cargo.project.RustSdkType
-
-class CargoInstallationManager {
-
-    fun hasCargoMetadata(cargoHomePath: String?): Boolean {
```

Comment:
Still I think it is a good strategy just to execute `metadata` and deal with possible failure.